src/rembg/cmd/cli.py

```diff
@@ -2,6 +2,7 @@
 import glob
 import imghdr
```

Comment:
✂️

Reply:
Thanks for your reminder.

Unused import is removed.